utils/spdx/src/test/kotlin/model/SpdxDocumentTest.kt

```diff
@@ -62,11 +52,12 @@ class SpdxDocumentTest : WordSpec({
     "The official YAML example without ranges from the SPDX specification version 2.2" should {
         "have idempotent (de)-serialization" {
             val yaml = readResourceAsText("/spdx-spec-examples/SPDXYAMLExample-2.2-no-ranges.spdx.yaml")
-
             val document = SpdxModelMapper.fromYaml<SpdxDocument>(yaml)
+
             val serializedYaml = SpdxModelMapper.toYaml(document)
+            val deserializedYaml = SpdxModelMapper.fromYaml<SpdxDocument>(serializedYaml)
 
-            serializedYaml shouldBe formatYaml(yaml)
+            deserializedYaml shouldBe document
```

Comment:
commit-message: Please indicate that the test no more asserts that all attributes present in the test asset are actually implemented.

Reply:
Maybe it's possible to achieve that be serializing both in alphabetical order (object mapper flag) and comparing the output, while reading the original file into a JsonNode tree, not into data class instances.  (I'm not saying it's worth the effort, I'm ok as-is).